### PR TITLE
Make event rules more visible

### PR DESCRIPTION
**Pull request by jennymul**
# Description

Remove how many spots there are left for an event and add concent for event rules

# Result

If you've made visual changes, please check the boxes below and include images showing the changes. Descriptions are appreciated.

- [X] Changes look good on both light and dark theme.
- [X] Changes look good with different viewports (mobile, tablet, etc.).
- [X] Changes look good with slower Internet connections.

<table>
    <tr>
        <td width="25%">Description</td>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            ...
        </td>
        <td>

        </td>
        <td>
        
        </td>
    </tr>
</table>

# Testing

- [X] I have thoroughly tested my changes.
---

Resolves ABA-276

### CODE REVIEW

File: app/routes/events/components/JoinEventForm.tsx [hunk 1]
```diff
@@ -172,24 +167,6 @@ const PaymentForm = ({
   </div>
 );
 
-const SpotsLeft = ({ activeCapacity, spotsLeft }: SpotsLeftProps) => {
```
Comment (ivarnakken) [inline app/routes/events/components/JoinEventForm.tsx:175]: In my opinion, we still need this. Any reason for removing?

Comment (jennymul) [reply app/routes/events/components/JoinEventForm.tsx:175]: Don't agree, everyone can do the simple math when it says 23/30 places left..

Comment (falbru) [reply app/routes/events/components/JoinEventForm.tsx:175]: It would also make the page more cluttered

**Review by falbru — APPROVED** [review]
Looks good!

File: app/routes/events/components/JoinEventForm.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/routes/events/components/JoinEventForm.tsx:175]: hmm okay, fair enough hehe

**Review by ivarnakken — APPROVED** [review]
Much more visible! Nice 😄

**Comment by ivarnakken** [comment]
> Also the mount() function, to my understanding, renders the element without the context of the rest of the route. It seems that the error message given in drone is related to the fact that the component uses <Link> without being wrapped in a router.

Yes this seems to be the issue. Wrapping the event form with a `MemoryRouter` seems to fix it though - at least on my computer™️ 

```diff
function renderJoinEventForm(props = {}) {
  return mount(
+    <MemoryRouter>
      <Provider store={store}>
        <JoinEventForm {...defaultProps} {...props} />
      </Provider>
+    </MemoryRouter>,
  );
}
```

**Pull request merged**